recipe-server/normandy/recipes/tests/test_api.py

```diff
@@ -679,11 +679,11 @@ def test_full_approval_flow(api_client):
         'enabled': 'false',
     })
     assert res.status_code == 201
-    recipe_data_1 = res.json()
+    recipe_data_0 = res.json()
```

Comment:
Semantic names for these variables, like `initial_recipe_api_data`, would make following this a bit easier.